docs/getting_started/models.md

```diff
@@ -15,6 +15,8 @@ Temporal Fusion Transformer<br>[Lim et al. 2021][Lim2021]    | Global       | Un
 Transformer<br>[Vaswani et al. 2017][Vaswani2017]            | Global       | Univariate               | MLP, multi-head attention | [MXNet][Transformer]
 WaveNet<br>[van den Oord et al. 2016][vanDenOord2016]        | Global       | Univariate               | Dilated convolution | [MXNet][WaveNet]
 SimpleFeedForward                                            | Global       | Univariate               | MLP | [MXNet][SFF_mx], [PyTorch][SFF_torch]
+DeepNPTS                                                     | Global       | Univariate               | MLP | [PyTorch][DeepNPTS_torch]
+MQF2<br>[Kan et al. 2022][Kan2022]                           | Global       | Univariate               | RNN, ICNN | [PyTorch][MQF2_torch]
```

Comment:
@lostella do you know if "Univariate" is correct for MQF2? Looks like this model is "multivariate" in time but actually univariate in target dim. 

cc @benidis